src/locales/sr/translation.json

```diff
@@ -126,6 +131,10 @@
     "thumbnailNavigation": "Сличице",
     "thumbnails": "Сличице",
     "toggleWindowSideBar": "Сајдбар",
+    "collection": "Колекција",
+    "totalCollections": "{{count}} колекција/е",
+    "totalManifests": "{{count}} објек(а)та",
+    "close": "Затворите",
```

Comment:
Is this `close` key needed here?

Reply:
Yes, it is on dialog panel, i suppose that this place is correct. Should it be in abecede direction?

Reply:
Thank you, yes please add in alphabetical direction!